AmplifyPlugins/API/AWSAPICategoryPlugin/Reachability/NetworkReachabilityNotifier.swift

```diff
@@ -36,13 +36,14 @@ class NetworkReachabilityNotifier {
         } catch {
             //TODO: Test effects of inability to start ReachabilitySwift
             Amplify.DataStore.log.error("Unable to start notifier from ReachabilitySwift")
+            throw error
         }
     }
 
     deinit {
         reachability?.stopNotifier()
         NotificationCenter.default.removeObserver(self)
-        reachabilityPublisher.send(completion: Subscribers.Completion<DataStoreError>.finished)
+        reachabilityPublisher.send(completion: Subscribers.Completion<Never>.finished)
```

Comment:
DataStoreError to APIError? or is Never correct?

Reply:
I believe Never is correct.  My thought was that if we can instantiate and start the reachability subsystem, then we do not expect to be sending any errors to the subscribers.